engine/access/rpc/backend/backend_test.go

```diff
@@ -87,6 +87,7 @@ func (suite *Suite) TestPing() {
 		metrics.NewNoopCollector(),
 		nil,
 		false,
+		100,
```

Comment:
Should we make this at least a constant in these test files? incase we want to adjust this for all tests

Reply:
@Kay-Zee Hah, beat me by a minute 😄